build/lib/extensions.js

```diff
@@ -226,6 +226,7 @@ const externalExtensions = [
     'sql-assessment',
     'sql-database-projects',
     'sql-migration',
+    'server-report'
```

Comment:
Would you mind putting this above the sql- ones to keep it in alphabetical order?